test/functional/services/dashboard/panel_settings.ts

```diff
@@ -64,6 +64,10 @@ export function DashboardCustomizePanelProvider({ getService, getPageObject }: F
           );
         }
       });
+
+      await retry.waitFor('superDatePickerToggleQuickMenuButton', async () => {
+        return await testSubjects.exists('superDatePickerToggleQuickMenuButton');
+      });
```

Comment:
We should maybe look for the testSubject inside the flyout element so it does not get confused with the time picker in the unified search bar. 

Maybe something like this that is also used in the `openDatePickerQuickMenu` method? 

```suggestion
      let button: WebElementWrapper | undefined;
      await retry.waitFor('superDatePickerToggleQuickMenuButton to be present', async () => {
        button = await this.findDatePickerQuickMenuButton();
        return Boolean(button);
      });
```